assets/dev/scss/frontend/general/_button.scss

```diff
@@ -25,13 +25,21 @@
 	}
 
 	&-icon {
+
+		i {
+			color: var(--button-icon-color, inherit);
+		    font-size: var(--button-icon-size);
+		}
+
 		svg {
-			width: 1em;
 			height: auto;
+			width: var(--button-icon-size, 1em);
+			fill: var(--button-icon-color, inherit);
 		}
 
 		.e-font-icon-svg {
 			height: 1em;
+			font-size: var(--button-icon-size);
```

Comment:
Do we need this line here?
Is it working?

What happens when you comment it out?